lib/index.js

```diff
@@ -77,15 +77,15 @@ SimpleReach.prototype.page = function(page) {
 SimpleReach.prototype.track = function(track) {
   var revenue = track.revenue();
   var orderId = track.orderId();
+  var eventName = track.event();
 
-  if (typeof revenue === 'number' && orderId) {
-    window.SPR.collect({
-      pid: this.options.pid,
-      reach_tracking: false,
-      url: track.proxy('context.page.url'),
-      title: track.proxy('context.page.title'),
-      ctx_revenue: revenue,
-      ctx_order_id: orderId
-    });
-  }
+  window.SPR.collect({
+    pid: this.options.pid,
+    reach_tracking: false,
+    url: track.proxy('context.page.url'),
+    title: track.proxy('context.page.title'),
+    ctx_revenue: revenue ? revenue : null,
```

Comment:
Just curious, do we need to send `ctx_revenue` or `ctx_order_id` if we do not have it? Because line 78 `track.revenue()` basically does this:

In order of precedence:
1) look up `properties.revenue`
2) if track `event` name is `Completed Order` or `Order Completed`, override and return `properties.total` instead
3) return `undefined`;

`track.orderId()` does the same except it doesn't do anything like step 2, just returns `properties.order_id || properties.orderId || undefined`

So we could make the code simpler here if we don't need to send these values without creating superfluous variables:

```js
window.SPR.collect({
   pid: this.options.pid,
   reach_tracking: false,
   url: track.proxy('context.page.url'),
   title: track.proxy('context.page.title'),
   ctx_revenue: track.revenue(),
   ctx_order_id: track.orderId(),
   ctx_event_name: track.event()
});
```